CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- **BREAKING** update `searchCivicPlusHCMSContentItems` to call HCMS directly
```

Comment:
This is not a breaking change to this package. Breaking (i.e. major semver bump) means that updating to the new package requires code changes.